### PR TITLE
Fix fixed mode camera speed

### PR DESCRIPTION
**Pull request by bd339**
Attempt at fixing #186 . My theory is that some configurations fire more mouse move events hence calling `directPan` more often, each time adding to `_angleH` and `_angleV`. At least that theory is consistent with the observations in [this comment](https://github.com/Senscape/Dagon/issues/186#issuecomment-270028839), so I'm going with it :smile:

The fix is to only call `directPan` when the camera updates, because that happens at a constant rate. Assuming linear growth of `directPan` (looks like it to me from the code) this does not change the general camera speed, because we would be reducing the number of `directPan`  calls by some factor, but increasing the observed mouse movement by the same factor.

Bottom line is, it works on my Linux test machine and didn't change the camera speed noticably on my Windows machine where fixed mode camera speed was normal.

Lots of whitespace noise in this patch because of the many lines with trailing whitespace that my editor cleaned up.

### CODE REVIEW

**Comment by bd339** [comment]
@civanT can you confirm if this also fixes the bug on your XP virtual machine? (Pull from the repo as usual, switch to iss186 branch, then build).

**Comment by bd339** [comment]
This does not fix the bug on Civan's virtual XP installation, but that sounds like an entirely different bug altogether (and we're only seeing it on his virtual XP installation so far). But this definitely solves my problem on Linux and it should apply to others as well.
